brainframe/cli/docker_compose.py

```diff
@@ -118,14 +119,13 @@ def _assert_has_docker_permissions():
         print_utils.fail(error_message)
 
 
-def _assert_has_write_permissions(path: Path):
+def _assert_has_write_permissions(path: Path) -> None:
     """Fails if the user does not have write access to the given path."""
     if os.access(path, os.W_OK):
         return
 
     error_message = i18n.t("general.file-bad-write-permissions", path=path)
-
-    print()
+    error_message += "\n"
```

Comment:
Seems that the newline is now printed _after_ the error message? Previously it would have been printed before. Is that intentional?

Reply:
This was actually a mistake on my part originally. My intention was to put a newline between the two parts of the error message.

Reply:
So the behavior now is how I originally wanted it.